lib/rage/router/dsl.rb

```diff
@@ -175,6 +175,24 @@ def defaults(defaults, &block)
       @defaults.pop
     end
 
+    def mount(*args)
+      if args.first.is_a?(Hash)
+        app = args.first.keys.first
+        at = args.first.values.first
+        via = args[0][:via]
+      else
+        app = args.first
+        at = args[1][:at]
+        via = args[1][:via]
+      end
+
+      # If app is a class, convert it to a lambda
+      app = ->(env) { app.new.call(env) } if app.is_a?(Class)
+
+      # Use match with via: :all to mount the Rack-based application
+      match(at, to: app, via:)
```

Comment:
Should it be something like `"#{at}/*"` in order to match all routes of the Rack application?

Reply:
good idea

Reply:
I don't think so, if the user want to do that he just can say:   mount Sidekiq::Web =>` '/sidekiq/*'`